packages/control-property-editor/src/components/ToolBar.tsx

```diff
@@ -1,9 +1,19 @@
-import type { ReactElement, PropsWithChildren } from 'react';
+import type { ReactElement } from 'react';
 import React from 'react';
 
-import { UIToolbar, UIToolbarColumn } from '@sap-ux/ui-components';
+import { UIFocusZone, UILabel, UIToolbar, UIToolbarColumn } from '@sap-ux/ui-components';
 
 import './ToolBar.scss';
+import { ViewChanger } from '../panels/properties/ViewChanger';
+import { DeviceSelector } from '../panels/properties/DeviceSelector';
```

Comment:
We shouldn't be importing anything from the `panels` folders here. Also toolbar shouldn't be in the `components` folder, because there are only supposed to be simple reusable components, that do not depend on redux state, which the toolbar does not fit. I'd suggest to create a folder `src/toolbar` folder for it and move all the related components there.